Makefile

```diff
@@ -1,4 +1,4 @@
-TAILSCALE_VERSION ?= "1.16.1"
+TAILSCALE_VERSION ?= "1.16.2"
 TAILSCALE_TRACK = "stable"
 # This needs to be monotinically increasing regardless of the TAILSCALE_VERSION
 SPK_BUILD = 13
```

Comment:
Doesn't SPK_BUILD need to be updated again? I thought it would have built another 1.16.1 release as SPK 13, with the logrotator fix that went in this morning.

Reply:
It doesn't do a release unless tagged